nixos/modules/programs/git-worktree-switcher.nix

```diff
@@ -0,0 +1,30 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+let
+  prg = config.programs;
+  cfg = prg.git-worktree-switcher;
+
+  initScript = ''
+    eval "$(command ${pkgs.git-worktree-switcher}/bin/wt init $(basename $SHELL))"
```

Comment:
```suggestion
  initScript =
    shell:
    if (shell != "fish") then
      ''
        eval $(${getExe pkgs.git-worktree-switcher} init ${shell})
      ''
    else
      ''
        ${getExe pkgs.git-worktree-switcher} init ${shell} | source
      '';```

Reply:
May I ask why would you want use 2 different scripts when we can just use `eval $(${getExe pkgs.git-worktree-switcher} init ${shell})` for everything?

Reply:
As fish is not a POSIX shell [`eval`](https://fishshell.com/docs/current/cmds/eval.html) isn't the intended way to load code into the shell, while `eval` may work in this case, let's please stick to piping into [`source`](https://fishshell.com/docs/current/cmds/source.html).

Reply:
Fair enough. I just used what upstream docs said as I'm not familiar with `fish`. I'll look into implementing this fix on my laptop shortly.